src/MultifilesLogger.jl

```diff
@@ -0,0 +1,138 @@
+struct FileDefForMultifilesLogger
+    filePath::String
+    append::Bool
+    modulesAndLogLevels::Vector{Tuple{Module,LogLevel}}
+end
+
+"""
+A multiple files logger for logging to files depending on the module.
+
+    fileDef1 = FileDefForMultifilesLogger("first.log",
+                                          true, # append
+                                          [(MyModule1.MySubModule1,Info),(MyModule1.MySubModule2,Info)],
+                                          )
+
+    fileDef2 = FileDefForMultifilesLogger("second.log",
+                                            true, # append
+                                            [(MyModule2,Info)],
+                                            )
+
+    fileDef3 = FileDefForMultifilesLogger("main.log",
+                                            true, # append
+                                            [(Main,Info)],
+                                            )
+
+    # Create the logger and set it as the global logger
+    multifilesLogger =
+        MultifilesLogger([fileDef1,fileDef2,fileDef3])
+
+    global_logger(multifilesLogger)
+
+
+"""
+struct MultifilesLogger <: _iologger
+    filesDefs::Vector{FileDefForMultifilesLogger}
+    logIOs::Dict{Module, Tuple{T,LogLevel}} where T <: IO
+    messageLimits::Dict{Any, Int}
+    flush::Bool
```

Comment:
Flushing is depending on the IO in question, not the logger itself.

Reply:
OK, I have moved the 'flush' property from 'MultifilesLogger' to 'FileDefForMultifilesLogger'. By default it is set to true.